api/src/main/java/org/pmiops/workbench/db/dao/UserService.java

```diff
@@ -151,7 +158,45 @@ public DbUser updateUserWithRetries(Function<DbUser, DbUser> userModifier, DbUse
     }
   }
 
-  private void updateDataAccessLevel(DbUser user) {
+  private void updateDataAccessLevel(DbUser dbUser) {
```

Comment:
This looks good for the bypass (administrative) methods. I don't think this is handling the "normal" path anymore though, e.g. `submitDataUseAgreement`, `syncComplianceStatus`, `syncEraCommons` etc

Reply:
Are you saying those should all funnel through the retry stuff? Or that they're broken now (in which case we likely have a unit test gap). I think I was thinking this was orthogonal to that.

Reply:
Sorry, nevermind - I misunderstood the goal of the PR. I see that updateUser() is still calling into this, so those should still function.

I mistakenly thought this change was also resolving RW-3830